site/pages/examples/[example].tsx

```diff
@@ -256,6 +267,7 @@ const ExamplePage = ({ example }: { example: string }) => {
   const [name, Component, path] = EXAMPLE
   return (
     <ErrorBoundary
+      FallbackComponent={ErrorFallback}
```

Comment:
Did you get a chance to manually test the result of an error here?

Reply:
Honestly no, but I just copy pasted this off the package Readme

Reply:
My assumption was this is not hit and I'd have to break one of the examples to hit this. Also given its test/examples code I was less concerned